spec/models/user_spec.rb

```diff
@@ -323,23 +323,6 @@ def provider_username(service_name)
         expect(user).not_to be_valid
       end
     end
-
-    context "when the past value is relevant" do
```

Comment:
Suggestion: move what this test is testing to a test for the User::Update service

Reply:
I added a test for setting `old_username` and `old_old_username` in `Users::Update` in my previous pr (https://github.com/forem/forem/blob/70e233953c4e8ecd98628aba4f30b8e82a5268e4/spec/services/users/update_spec.rb#L80-L88)